src/shared/components/palette/dynamic-list/write-dynamic-list-field.component.ts

```diff
@@ -23,8 +23,10 @@ export class WriteDynamicListFieldComponent extends AbstractFieldWriteComponent
      *
      * Reassigning value from formatted_value when value is empty
      */
-    if (!this.caseField.value && this.caseField.formatted_value && this.caseField.formatted_value.value && this.caseField.formatted_value.value.code) {
-      this.caseField.value = this.caseField.formatted_value.value.code;
+    if (!this.caseField.value) {
+      if (this.caseField.formatted_value && this.caseField.formatted_value.value && this.caseField.formatted_value.value.code) {
+        this.caseField.value = this.caseField.formatted_value.value.code;
+      }
     }
 
     let isNull = this.caseField.value === undefined || this.caseField.value === '';
```

Comment:
I was wondering whether to `let` you off on this one but changed my mind 😃

Reply:
Not my code but good spot. 😂

Reply:
Done.